.github/workflows/s3Deploy.yml

```diff
@@ -0,0 +1,35 @@
+name: Deploy to S3
+
+on:
+  push:
+    branches:
+      - main
+
+# Required for authenticating with AWS IAM
+permissions:
+  id-token: write
+  contents: read
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Configure AWS credentials
+        uses: aws-actions/configure-aws-credentials@v2
+        with:
+          role-to-assume: arn:aws:iam::693696263829:role/token-list-github-action-role
+          role-session-name: githubactionsession
+          aws-region: eu-central-1
+
+      - name: Upload to S3
+        run: |
+          aws s3 cp src/public/CowSwap.json s3://files.cow.fi/tokens/CowSwap.json
+
+      - name: Invalidate CloudFront cache
+        run: |
+          aws cloudfront create-invalidation \
+            --distribution-id ${{ secrets.DISTRIBUTION }} \
+            --paths "/tokens/*"
```

Comment:
cool! 

Is there a way to verify this does what we want? 
Maybe, we can have it in the check-list on what to verify once we do our first release? (would we need to check anything in AWS or we should check sth in the headers after deployment)

Reply:
Agreed

Reply:
We can check this from the AWS CloudFront console. We can also check last modified and etag response from CloudFront.